doc/neomake.txt

```diff
@@ -88,12 +88,16 @@ All 'args' will be |expand()|ed: >
         \ 'args': ['%:p'],
         \ }
 
-The 'exe' and 'args' entries can be a function (callback) or a dictionary with
-an expected entry 'fn' (which has to be a function).
-The function has to return a string, and receives no arguments.
+The 'exe' and 'args' entries can be a function reference (see |:h Funcref|) or
+a dictionary with an expected entry 'fn' (which has to be a function).
+The function must return a string, and the function will be called with no
+arguments.
 
-Running "|:Neomake!| lint" from file.txt would cause "lint /path/to/file.txt" to
-be run.
+In the above example, running "|:Neomake!| lint" from file.txt would cause
+"lint /path/to/file.txt" to be run. Do note that this is atypical usage and is
+provided as an example of args expansion only, since "|:Neomake!|" commands
+typically don't require the filename. The preferred method of running Neomake
+against the current file is "|:Neomake|" without the exclamation mark.
```

Comment:
Well, this somehow manifests the arg usage in the doc.
And it's not really "args expansion" but using "lint" as the maker, isn't it?